build.sbt

```diff
@@ -102,17 +101,23 @@ lazy val `scalatestplus-play` = project
     mimaSettings,
     organization := "org.scalatestplus.play",
     libraryDependencies ++= Seq(
+      // Note: It seems like the only jackson version that works with all play, play-ws, play-json and selenium is 2.11.4.
+      //       play will bring in older version of selenium as well, so need excluding and let scalatest+selenium brings in the newer version.4
```

Comment:
If I remember correctly, this is because new selenium versions do not support Java 8 anymore, but right now Play still is build with Java 8.
However we are going to drop Java 8 very soon, so that would resolve that problem.